Tabloid/Controllers/PostController.cs

```diff
@@ -25,6 +25,17 @@ public IActionResult Get()
             return Ok(_postRepository.GetAll());
         }
 
+        [HttpGet("{id}")]
+        public IActionResult Post(int id)
```

Comment:
Naming this method "Post" is  a bit confusing, seeing as it's a GET method.  Is there something else you could name it that would indicated what it's doing?